ascii art acknowledgements (#53)
rats-endorsements-distribution/draft-howard-rats-coserv: draft-howard-rats-coserv.md

# Acknowledgments
{:numbered="false"}

> The participants to the "Staircase meeting" at FOSDEM 25:

Reviewer: This should be "in" or "at" or "of" rather than "to".

Author: Fixed in [dd611a6](https://github.com/rats-endorsements-distribution/draft-howard-rats-coserv/pull/53/commits/dd611a63ef6417d55e26482de246144352defbd0), thanks!

> The participants to the "Staircase meeting" at FOSDEM 25:

Author: ```suggestion
The participants in the "Staircase meeting" at FOSDEM 25:
```

%===++*++#***#@**%%%#%#+==========================%%%==================
######*****###*##*##===========================%%%#====================
#######****##*#*##**=====+++=++-----=======++%%#+======================
#########**#***##*#*#=+=======-:::::--====@%##-========================
######*=####**=#####*++++++++=-------+++@%##--=============------------
%%######@@@@@@@%++++=%====%+==----=++%%%#==---=========================
%%%%#%%###@@=**=++=------==#----+++%%%#+====-======================+===
======+##=%@++**=#-:--::---+-+++%%%#===== Ionut Mihalcea ==============
---------=+=*++%#*+=-=::---*++%%%#==================================+==
------------==%==-:::=*=*##%%%%#======*================================
===========-=##------+%%@%%%#+====================== =  /` _____ `\;,
###############*+*****@%%%#+========================   /__(^===^)__\';,
*############++++++#%%%#++==========++==============     /  :::  \   ,;
+++****##*#++=+++@%%%#+==== The Staircase ==========    |   :::   | ,;'
++++++++======++@#%#================================    '._______.'`
++++++++====++@#%#===================================  Dionna Glaze ===
~~~

> For mysterious reasons Henk Birkholz and Jag Raman are not in the frame, but they were there with us!

Reviewer 2: ```suggestion
For mysterious reasons Henk Birkholz and Jag Raman are puppeteering in the shadows.
```
